admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/component/AdminConfirmationDialog.kt

```diff
@@ -0,0 +1,117 @@
+package net.thechance.mena.admin_panel.presentation.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.unit.dp
+import net.thechance.mena.admin_panel.resources.Res
+import net.thechance.mena.admin_panel.resources.cancel
+import net.thechance.mena.designsystem.presentation.component.button.OutlinedButton
+import net.thechance.mena.designsystem.presentation.component.button.PrimaryButton
+import net.thechance.mena.designsystem.presentation.component.dialog.BasicDialog
+import net.thechance.mena.designsystem.presentation.component.icon.Icon
+import net.thechance.mena.designsystem.presentation.component.scaffold.ScaffoldScope
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import org.jetbrains.compose.resources.stringResource
+
+@Composable
+fun ScaffoldScope.AdminConfirmationDialog(
+    dialogIcon: Painter,
+    confirmationIcon: Painter,
+    title: String,
+    description: String,
+    confirmationButtonText: String,
+    isVisible: Boolean,
+    onDismiss: () -> Unit,
+    onConfirm: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    BasicDialog(
+        onDismiss = onDismiss,
+        isVisible = isVisible,
+        dialogCornerShape = RoundedCornerShape(24.dp),
+        contentPadding = PaddingValues(24.dp),
+        hasDismissButton = false,
+        contentColor = Theme.colorScheme.background.surfaceLow,
+        actionButtons = {
+            Row(
+                modifier = Modifier
+                    .padding(top = 12.dp)
+                    .fillMaxWidth(),
+                horizontalArrangement = Arrangement.spacedBy(8.dp)
+            ) {
+                PrimaryButton(
+                    text = stringResource(Res.string.cancel),
+                    onClick = onDismiss,
+                    contentPadding = PaddingValues(
+                        start = 16.dp,
+                        end = 16.dp,
+                        top = 8.dp,
+                        bottom = 8.dp
+                    )
+                )
```

Comment:
Check the design again, same for the other button
```suggestion
                    contentPadding = PaddingValues(
                        horizontal = 16.dp,vertical = 12.dp,
                    )
              
```